docs/sources/datasources/prometheus/configure-prometheus-data-source.md

```diff
@@ -99,15 +95,21 @@ Use TLS (Transport Layer Security) for an additional layer of security when work
 
 - **Value** - The value of the header.
 
-## Additional settings
+## Advanced settings
 
 Following are additional configuration options.
 
+### Advanced HTTP settings
+
+- **Allowed cookies** - Specify cookies by name that should be forwarded to the data source. The Grafana proxy deletes all forwarded cookies by default.
+
+- **Timeout** - The HTTP request timeout. This must be in seconds. There is no default, so this setting is up to you.
+
 ### Alerting
 
 - **Manage alerts via Alerting UI** - Toggle to enable `Alertmanager` integration for this data source.
 
-### Interval behavior
+### Interval behaviour
```

Comment:
We use US English in documentation:https://grafana.com/docs/writers-toolkit/write/style-guide/inclusive-writing/#write-for-an-international-audience

```suggestion
### Interval behavior
```